src/content/CloudBoilerplateNet/Services/WebhookListener.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+using CloudBoilerplateNet.Models;
+
+namespace CloudBoilerplateNet.Services
+{
+    public class WebhookListener : IWebhookListener
```

Comment:
probably shouldn't be under services...consider refactoring the folder structure a bit

Reply:
GP! Will do.